src/components/NewsRoomCard.jsx

```diff
@@ -37,9 +37,10 @@ const NewsRoomCard = props => {
               thumbnailAltText={thumbnailAltText}
             />
           </div>
-          <div className="col-md-9 col-xs-12">
+          <div className="col-md-9 col-xs-12 dg_news_date-author">
```

Comment:
i feel like this name could be improved for resuse, but I don't have a good suggestion. Author to me makes me think id only find Author stuff, but im not sure i can think of a better name.